src/DebugChannel.js

```diff
@@ -42,14 +44,36 @@ export class DebugChannel {
   /* Request object based on the protocol
   */
   readIn(): string {
-    let contents = "";
-    while (contents.length === 0) {
-      contents = this.fs.readFileSync(this.inFilePath, "utf8");
+    let message = "";
+    while (true) {
+      let contents = this.fs.readFileSync(this.inFilePath, "utf8");
```

Comment:
Can you prefix all the private fields(inFilePath, fs, etc...) with leading underscore?